includes/class-coblocks-register-blocks.php

```diff
@@ -93,14 +93,6 @@ public function register_blocks() {
 				'style'         => $slug . '-frontend',
 			)
 		);
-		register_block_type(
-			$slug . '/click-to-tweet',
-			array(
-				'editor_script' => $slug . '-editor',
-				'editor_style'  => $slug . '-editor',
-				'style'         => $slug . '-frontend',
-			)
-		);
```

Comment:
This while file surprised me. I somehow forgot that this file exists but honestly, I wonder why it exists at all. There are only 12 blocks registered here some of which are JavaScript blocks that are registered in JS. There are SSR blocks like the Form block that is not registered here at all but separately within its respective index file.

In fact, I do not see this class used at all outside of a test written for it.

Reply:
I was mistaken on this. The file is invoked in class-coblocks.php. Either way, we still have a question of why some SSR blocks are registered elsewhere and why JS blocks are registered here at all. Does superfluous registration have performance impacts for instance? Regardless these are things that should be handled in a different story.